docker/data-science/notebooks/semantic_search/query_tree.py

```diff
@@ -56,17 +55,19 @@ def process_semantic_functions(query_tree):
 
     return query_tree 
 
-def get_enrichments(collection, keyword):
+def get_enrichments(collection_name, keyword):
```

Comment:
Not understanding why the change from `collection` to `collection_name`. Seems using the object would preserve the abstraction better, expecting a collection object (with the name) to exist already and be passed in.